### ISSUE TITLE
Controlling tabindex focus upon re-render

### ISSUE THREAD

**Issue by jayhlee** [putyourlightson/craft-sprig#155]
### Question
Is there a way to control the location of the tabindex focus after a Sprig render? 

For example, if I have three Sprigified tab links, I would like the tabindex focus to remain on the clicked tab link. This is especially important for accessibility use cases where a user tabs to a specific tab (the tab receives keyboard focus) and hits Return. Instead of the tab focus remaining on the activated tab link, the tab focus seems to lock focus on links in my footer. It even skips the links from the returned, rerendered content—a bit mysterious and bizarre.

### Additional context
The tabindex focus seems to work as it should with dropdown (and checkbox fields). When you keyboard select a dropdown option, the focus still remains on the same dropdown field after a Sprig re-render. 

In short, I'm trying to replicate the same behavior with non-dropdown links.

**Comment by bencroker** [comment]
I believe that the focus is maintained for elements that have IDs, but you'll need to ask over in https://github.com/bigskysoftware/htmx/issues since this is a htmx issue.
